huawei_solar_battery_optimization.yaml

```diff
@@ -502,7 +504,7 @@ script:
       - service: huawei_solar.set_tou_periods
         data:
           periods: 00:00-00:01/1234567/+
-          device_id: 8ee3ed713a270c3bbf9596bcfe414175
+          device_id: "{{ states('input_text.hsbo_device_id') }}"
```

Comment:
this one should be

device_id: "{{ states('input_text.hsbo_device_id_batteries') }}"